.github/workflows/crlf-check.yaml

```diff
@@ -0,0 +1,18 @@
+name: Check Line Endings (find CRLF)
+
+on: 
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  check_pull_request_job:
+    runs-on: ubuntu-latest
+    name: Check files for CRLF line endings
+    steps:
+    - name: Checkout repository contents
+      uses: actions/checkout@v1
+
+    - name: Validate files
+      run: |
+        chmod +x "./.github/crlf-check.sh"
```

Comment:
update the file to have the executable flag and commit that so this line can be removed.
```suggestion
```

`git update-index --chmod=+x path/to/file`
https://git-scm.com/docs/git-update-index#Documentation/git-update-index.txt---chmod-x

Reply:
Replaced custom code with github action